tests/components/webostv/test_config_flow.py

```diff
@@ -43,6 +52,16 @@ async def test_form_import(hass, client):
     assert result["type"] == RESULT_TYPE_CREATE_ENTRY
     assert result["title"] == "fake"
```

Comment:
Should we assert the result data and created config entry unique_id here too?

Reply:
We should